src/test/vstart_wrapper.sh

```diff
@@ -16,6 +16,7 @@
 # GNU Library Public License for more details.
 #
 
+CEPH_ROOT=${CEPH_ROOT:-..}
```

Comment:
@alimaredia 
I suggested this under the assumption that this is run from the `....../src` directory.
And then the way to `qa/`is just `..`
If you want to run it from the root of the whole ceph-directory, you'd need `../..`
What you would not do is run it whilest standing in src/test.

So in that sense it could do with a lot more code to actually determine where `$CEPHOOT` should be. But it could be done with a range of test to see where `./qa` actually is.